bitnami/keycloak/values.yaml

```diff
@@ -35,6 +35,12 @@ namespaceOverride: ""
 ## @param commonLabels Labels to add to all deployed objects
 ##
 commonLabels: {}
+## @param serviceLinks Boolean to disable kubernetes service links
+##
+serviceLinks:
+  ## @param serviceLinks.disabled Disable kubernetes service links (enabled by default in kubernetes)
+  ##
+  disabled: false
```

Comment:
Thanks for your contribution, there is an integration that is failing due to an issue in the metadata used to generate the README. Could you apply something like this?
```suggestion
## @param serviceLinks.disabled Disable kubernetes service links (enabled by default in kubernetes)
##
serviceLinks:
  disabled: false
```